Review Assignment/src/assets/scripts/about.js

```diff
@@ -0,0 +1,162 @@
+window.about = function() {
+  var aboutPage = {};
+  var type = 'GET';
```

Comment:
Create a separate method in common file where you keep all method type as constants.